CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+### v0.20.1 - 2022/02/08
+##### Bug fixes
+* HTTP compression always on and max retries to not exceed 15 mins in the ES client [#69](https://github.com/elastic/elastic-serverless-forwarder/pull/69)
```

Comment:
I feel like a verb is missing, maybe you meant `Set HTTP compression...`?

Reply:
indeed: explicit is better than implicit :)